hydra-node/test/Hydra/Model.hs

```diff
@@ -154,6 +155,7 @@ instance StateModel WorldState where
     -- different return values.
     Init :: Party -> Action WorldState ()
     Commit :: Party -> UTxOType Payment -> Action WorldState ActualCommitted
+    Decommit :: Party -> Payment -> Action WorldState ()
```

Comment:
Modelling a decommit to also just select a `UTxOType Payment` to remove could be simpler?